src/Keboola/ThoughtSpot/Writer.php

```diff
@@ -28,12 +32,23 @@ public function __construct($dbParams, Logger $logger)
 
         parent::__construct($dbParams, $logger);
 
-        $this->checkDatabaseExists($dbParams);
-
-        $this->checkSchemaExists($dbParams);
+        try {
+            $this->checkDatabaseExists($dbParams);
+        } catch (DatabaseNotExistsException $exception) {
+            if (!$this->createDatabase($dbParams)) {
+                throw new UserException($exception->getMessage(), 0, $exception);
+            }
+        }
+        try {
+            $this->checkSchemaExists($dbParams);
+        } catch (SchemaNotExistsException $exception) {
+            if (!$this->createSchema($dbParams)) {
+                throw new UserException($exception->getMessage(), 0, $exception);
```

Comment:
Ta UserException ma message z ty predosly `SchemaNotExistsException`? Nemelo by tam spis byt "Unable to create schema"? 
To samy u ty predosly `DatabaseNotExistsException `.

Reply:
🤦‍♂ máš pravdu - opraveno

Reply:
Som si myslel, ze si to nespravil schvalne :)